e2e/deployers/disapp.go

```diff
@@ -50,15 +50,7 @@ func (d DiscoveredApp) Deploy(ctx types.TestContext) error {
 		return err
```

Comment:
Since we're consolidating kustomization functionality in `util/kustomize.go`, i think we should move `CreateKustomizationFile` there as well, in a separate commit though.